pkg/sdk/connections_dto_gen.go

```diff
@@ -38,11 +38,11 @@ type ToAccountsRequest struct {
 	Accounts []AccountIdentifier
 }
 
-type SetRequest struct {
+type ConnectionSetRequest struct {
```

Comment:
This change looks good, but it's missing from `def` file. Was this file not generated properly, or `def` file should be adjusted?

Reply:
def should be adjusted